worker/deploy/README.md

```diff
@@ -1,47 +1,123 @@
-# Worker Node deployment
+# Worker Node Deployment
 
-Arrebol tasks are processed by the Workers. In a typical deployment, multiple Workers are deployed together in a single virtual machine, a `Worker Node`. As each task run in an isolated docker container, the Docker engine and other dependencies must have be installed and configured in the Worker Node. Below, we detail the configuration procedure.
+## **Overview**
 
-## Requeriments
+### _Worker Node_
 
-Before the configuration and installation of Worker Node dependencies, each Worker Node virtual machine should be configured to be reached via SSH (using a rsa key pair). Also, the [Ansible](https://www.ansible.com/) automation tool should be installed in the deploy `coordination host`. This host is a machine that commands that deploy. To this end, the `coordination host` should be able to access the Worker Node through SSH.
+**Worker Node** is the host where workers live. You need to deploy at least one worker node for Arrebol.
 
-To install Ansible run these commands:
+### _Coordination Host_
+
+The **coordination host** is basically a machine from which scripts are executed to deploy worker nodes. The coordination host must have access to worker nodes via **SSH (using a rsa key pair)** to perform its function.
```

Comment:
is a machine from which the worker nodes deployments are done

Reply:
Done.